packages/amplify-cli/src/__tests__/commands/uninstall.pkg.test.ts

```diff
@@ -60,6 +60,18 @@ describe('uninstall packaged CLI on mac / linux', () => {
       value: originalPlatform,
     });
   });
+
+  expect.addSnapshotSerializer({
+    test(val) {
+      return typeof val === 'string';
+    },
+    print(val) {
+      if (typeof val === 'string')
+        return `"${val.replace(/\\/ig, '/')}"`;
```

Comment:
We shouldn't convert the slashes to forward slashes. We should convert to the proper number Windows slashes.